recipes/python-rocksdb/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "python-rocksdb" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aa692fbd06d8ed4ad641feee2970062fe01120e87b645a29403c71c628639e4f
+
+build:
+  preserve_egg_dir: 'True'
+  number: 0
+  missing_dso_whitelist:
+    - '*/libc.so.6'
+
+requirements:
+  build:
+    - python
+    - setuptools >=25
+    - rocksdb >=6.10.1
+    - bzip2 >=1.0.8
+    - Cython >=0.20
+    - zlib >=1.2.11
+
+  host:
+    - rocksdb >=6.10.1
+    - python
+    - pip >=20.1.1
+    - Cython >=0.20
+    - bzip2 >=1.0.8
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
It seems like build and host are switched

Reply:
Indeed, fixed###